server/repository/src/migrations/v1_07_00/invoice_add_currency_fields.rs

```diff
@@ -9,55 +9,7 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
         r#"
         ALTER TABLE invoice ADD COLUMN currency_id TEXT REFERENCES currency(id);
         ALTER TABLE invoice ADD COLUMN currency_rate {DOUBLE};
-        ALTER TABLE invoice_line ADD COLUMN foreign_currency_price_before_tax {DOUBLE};
```

Comment:
Moved this to the invoice_line_add_item_link_id so that the view is only recreated once